src/client.rs

```diff
@@ -120,14 +87,18 @@ impl Client {
     ///   assert!(!buffer.is_empty());
     ///}
     /// ```
-    pub async fn get<I, K, V>(&self, path: &str, params: Option<I>) -> Result<reqwest::Response>
+    pub async fn get<I, K, V>(
+        &self,
+        path: &str,
+        params: Option<I>,
+    ) -> reqwest::Result<reqwest::Response>
     where
         I: IntoIterator,
         I::Item: Borrow<(K, V)>,
         K: AsRef<str>,
         V: AsRef<str>,
     {
-        let mut url = Url::parse(&format!("https://{}{}", super::API_HOST, path))?;
+        let mut url = Url::parse(&format!("https://{}{}", super::API_HOST, path)).unwrap();
```

Comment:
We should never unwrap `Result`s in library code.
Is there a reason for the changed method signature (`reqwest::Result` instead of `Result`)?

Reply:
Was trying to minimize the sum total of changes in this pull request. Adding it back.